src/index.ts

```diff
@@ -234,6 +237,28 @@ export class MongoLogWriter extends Writable {
     this.write(logEntry);
   }
 
+  /**
+   * Write a log entry with severity 'D'.
+   */
+  debug(
+    component: string,
+    id: MongoLogId,
+    context: string,
+    message: string,
+    attr?: unknown,
+    level: 1 | 2 | 3 | 4 | 5 = 1
+  ): void {
```

Comment:
This is missing in `bindComponent` now, right?

Reply:
Yes, totally missed that, thanks!